CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 Canonical reference for changes, improvements, and bugfixes for Boundary.
 
-## Next
+## 0.19.2 (2025/04/28)
```

Comment:
Is the planned release date for this 28th of April? Usually we'll stage a week or so ahead of the planned release date. You can check with Katrina what the planned release date is, but we can also update this afterwards, it's no biggie.